cloudslang-entities/src/main/java/io/cloudslang/lang/entities/PromptType.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * (c) Copyright 2016 Hewlett-Packard Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *******************************************************************************/
+package io.cloudslang.lang.entities;
+
+import java.util.Objects;
+
+public enum PromptType {
+    TEXT("text"), SINGLE_CHOICE("single_choice"), MULTI_CHOICE("multi_choice");
```

Comment:
should be _single-choice_ and _multi-choice_

Reply:
Changed as requested